docs/tutorials/cloud-agent.md

```diff
@@ -0,0 +1,160 @@
+
+- A Prefect Cloud [`PREFECT_API_KEY`]/concepts/settings/#prefect_api_key) value specifying the API key used to authenticate with your Prefect Cloud workspace. (Prefect Cloud organizations may use a [service account](/ui/service-accounts/) API key.)
+- The [`PREFECT_API_URL`](/concepts/settings/#prefect_api_url) value specifying the API endpoint of your Prefect Cloud workspace.
+
+`--command-line` lets you override the container’s normal entry point and run a command instead. The script above uses this section to install the `adlfs` pip package so it can read flow code from Azure Blob Storage, along with `s3fs`, `pandas`, and `requests`. It then runs the Prefect agent, in this case using the default work pool and a `test` work queue. If you want to use a different work pool or queue, make sure to change these values appropriately.
+
+## Create a deployment
+
+Following the example of the [Flow deployments](/tutorials/deployments/) tutorial, let's create a deployment that can be executed by the agent on this container instance. You'll also need a storage block configured to save your flow code as described in the [Storage and Infrastructure](/tutorials/storage/) tutorial.
+
+In an environment where you have [installed Prefect](/getting-started/installation/), create a new folder called `health_test`, an within it create a new file called `health_flow.py` containing the following code.
+
+
+```python
+import prefect
+from prefect import task, flow
+from prefect import get_run_logger
+
+
+@task
+def say_hi():
+    logger = get_run_logger()
+    logger.info("Hello from the Health Check Flow! 👋")
+
+
+@task
+def log_platform_info():
+    import platform
+    import sys
+    from prefect.server.api.server import SERVER_API_VERSION
+
+    logger = get_run_logger()
+    logger.info("Host's network name = %s", platform.node())
+    logger.info("Python version = %s", platform.python_version())
+    logger.info("Platform information (instance type) = %s ", platform.platform())
+    logger.info("OS/Arch = %s/%s", sys.platform, platform.machine())
+    logger.info("Prefect Version = %s 🚀", prefect.__version__)
+    logger.info("Prefect API Version = %s", SERVER_API_VERSION)
+
+
+@flow(name="Health Check Flow")
+def health_check_flow():
+    hi = say_hi()
+    log_platform_info(wait_for=[hi])
+```
+
+Now create a deployment for this flow script, making sure that it's configured to use the `test` queue on the `default-agent-pool` work pool.
+
+<div class='terminal'>
+```bash
+prefect deployment build --infra process --storage-block azure/flowsville/health_test --name health-test --pool default-agent-pool --work-queue test --apply health_flow.py:health_check_flow
```

Comment:
Users are likely going to get tripped up on using storage blocks. What do you think about including (or linking to) and example of setting up Azure storage? Also, I thought we only allowed lowercase letters and dashes in block names?

Reply:
`health_test` is a sub-folder name passed to the block for this deployment.

I'll take another look at how we've handled blocks in previous tutorials.